File: Python/diagonalDiference.py
#!/bin/python3

import math
import os
import random
import re
import sys

#
# Complete the 'diagonalDifference' function below.
#
# The function is expected to return an INTEGER.
# The function accepts 2D_INTEGER_ARRAY arr as parameter.
#

def diagonalDifference(arr):
    primary = 0
    secondary = 0
    aux = len(arr[0])

    for i in range(aux):
        primary += arr[i][i]
        secondary += arr[i][aux - i - 1]
    
    return abs(secondary-primary)

if __name__ == '__main__':
    fptr = sys.stdout 

    n = int(input().strip())

    arr = []

    for _ in range(n):
        arr.append(list(map(int, input().rstrip().split())))

    result = diagonalDifference(arr)

    fptr.write(str(result) + '\n')

    fptr.close()

File: JavaScript/diagonalDiference.js
function diagonalDifference(arr) {
    let primary = 0
    let secondary = 0
    let aux = arr[0].lenght

    for (let index = 0; index < aux; index++) {
        primary += arr[index][index]
        primary += arr[index][aux - i - 1]
    }

    return secondary - primary
}

const a = [[11,2,4],[4,5,6],[10,8,-12]]
console.log(diagonalDifference(a))